pages/vagrant.md

```diff
@@ -77,6 +77,13 @@ Additional subcommands are available, but are either more advanced
 or not commonly used. To see all subcommands, run the command
 `vagrant list-commands`.
 ```
+
+## Troubleshooting
+When typing the command "vagrant up" in OSX, you may experience an error such as the following: "vi: Box 'ole/jessie64' could not be found. Attempting to find and install...". A simple solution is to perform the command below. ```sudo rm /opt/vagrant/embedded/bin/curl``` This will remove the old version of curl and vagrant should now work as usual.
+
+For more information go here: (http://stackoverflow.com/questions/23874260/error-when-trying-vagrant-up)
```

Comment:
Thanks for the suggestions. I have added the proper markdown URL and also fixed the header formatting.